backend/src/loaders/lockdown/lockdown.js

```diff
@@ -176,39 +162,51 @@ function getEntry(sheet, entryIndex) {
  */
 export async function batchGetTerritoriesEntryData(territories) {
   const batchSize = 25;
-  const entriesToGrab = 10;
-  const endCacheColumn = columnToLetter(letterToColumn('H') + (entriesToGrab * entryColumnLength));
-  const rangeToCache = `H1:${endCacheColumn}60`;
   const doc = await getDocument();
+  // TODO: Figure out how to find total columns should take
+  const entriesToGrab = 1000;
+  const startColumnIndex = letterToColumn('D');
+  const endCacheColumn = columnToLetter(startColumnIndex + entriesToGrab);
+  const rangeToCache = `D1:${endCacheColumn}65`;
   const result = [];
   var batch;
   
   while (batch = territories.splice(0, batchSize)) {
     if (batch.length < 1) break;
     // TODO: Uncomment the following when country tab sheets are ready with ISO3 naming
-    // let gridRanges = batch.map(territory => `${territory['iso3']}!${rangeToCache}`);
-    let gridRanges = batch.map(territory => `DEMO!${rangeToCache}`);
+    // let gridRanges = batch.map(territory => `${territory['iso3']}_Fetch!${rangeToCache}`);
+    let gridRanges = batch.map(territory => `ASM_Fetch!${rangeToCache}`);
     logger.log(`[Lockdown:WorkSheet] ${batch.map(t => t['iso3']).join(' ')}`);
     let gridData = await doc.batchGetGridRanges(gridRanges);
     
     for (let i = 0; i < batch.length; i++) {
       // TODO: Uncomment the following when country tab sheets are ready with ISO3 naming
-      // let workSheet = await getWorksheetByTitle(batch[i]['iso3']);
-      let workSheet = await getWorksheetByTitle('DEMO');
-      let gridSheet = new SimpleGridSheet(rangeToCache, gridData[i], workSheet);
+      // let workSheet = await getWorksheetByTitle(`${batch[i]['iso3']}_Fetch`);
+      let workSheet = await getWorksheetByTitle('ASM_Fetch');
```

Comment:
Using await in a loop makes the async processing sequential, while we could parallelize this work. is this intentional?

Reply:
`getWorksheetByTitle` is a lazy-loaded function. The intention of using async was just for code readability, parallelizing this wouldn't make much of an impact. Most intensive operations are between these [lines](https://github.com/Code-for-All/lockdown/pull/129/files#diff-4c1a2a4721c4db3c9e706007353b0ce2R192-R198)